dova/www/templates/quizPage.html

```diff
@@ -1,4 +1,4 @@
-<ion-view title="Quiz page" id="page8">
```

Comment:
QuizPage and another thing are both no longer used, would you mind cleaning them up?